rules/falco_rules.yaml

```diff
@@ -998,11 +998,11 @@
     and privilege escalation (CVE-2020-14386) by an attacker. Noise can be reduced by using the user_known_packet_socket_binaries
     template list.
   condition: > 
-    evt.type=socket 
+    evt.type=socket and evt.dir=>
```

Comment:
`evt.arg.domain` is only defined for the enter event

Reply:
great catch